classes/03class/exercises/c03-docker05/mrcsmonteiro/README.md

```diff
@@ -0,0 +1,51 @@
+# C03-Docker05
+
+## Command Execution Output
+- Commands executed to run both containers:
+```
+$ docker pull mrcsmonteiro/exercise02:v1
+v1: Pulling from mrcsmonteiro/exercise02
+aad63a933944: Pull complete
+29ade582b51e: Pull complete
+7e41ad5b6f9c: Pull complete
+ebf61b47b4ca: Pull complete
+9c060bce4eae: Pull complete
+379d5a58a935: Pull complete
+Digest: sha256:d7ebc4eeaded7f0f8fdb7ac53ad90fafa552c94a5b66c35eb8d89e7404e96efa
+Status: Downloaded newer image for mrcsmonteiro/exercise02:v1
+docker.io/mrcsmonteiro/exercise02:v1
+
+$ docker pull mrcsmonteiro/exercise02:v2
+v2: Pulling from mrcsmonteiro/exercise02
+aad63a933944: Already exists
+29ade582b51e: Already exists
+7e41ad5b6f9c: Already exists
+ebf61b47b4ca: Already exists
+9c060bce4eae: Already exists
+a70e583a44c8: Pull complete
+Digest: sha256:84b4bb4c993c028645870afa77cd23e83db77736c589dde17cd7b3b3427faebf
+Status: Downloaded newer image for mrcsmonteiro/exercise02:v2
+docker.io/mrcsmonteiro/exercise02:v2
+
+$ docker run -dit -p 8081:80 mrcsmonteiro/exercise02:v1
```

Comment:
what does the `-dit` means in this execution?

Reply:
It has a special meaning, @kikobr82 : unintended copy/paste 😝 - Fixed.